timetracker/timetracker/urls.py

```diff
@@ -15,11 +15,24 @@
 """
 from django.conf.urls import include, url
 from django.contrib import admin
-from django.http import HttpResponse
+from django.http import HttpResponse, HttpRequest
 
 
 def hello_world(request):
-    return HttpResponse('Hello World')
+	visitorName = request.GET.get("name", False)
+	responseString = ""
+	if visitorName == False:
```

Comment:
When you're looking at true/false values, you don't need the `== False` here.

You could simple do:

```
if not visitorName:
```

To achieve the same thing.